.github/workflows/python-package.yml

```
@@ -33,12 +33,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13', 'pypy-3.10']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13', “3.14-dev“, 'pypy-3.11']
```

```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13', “3.14“, 'pypy-3.11']
```
This is no longer needed with `allow-prereleases: true` below.